CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - Added `removeBlur()` method for removing the applied blur effect from the view. [#1159](https://github.com/SwifterSwift/SwifterSwift/pull/1159) by [regi93](https://github.com/regi93)
   - Added `makeCircle(diameter:)` method to make the view circular. [#1165](https://github.com/SwifterSwift/SwifterSwift/pull/1165) by [happyduck-git](https://github.com/happyduck-git)
 
+### Changed
```

Comment:
Move up to new release

Reply:
Updated branch to keep up-to-date with master